docs/install/installing-istio.md

```diff
@@ -20,6 +20,12 @@ You need:
 - A Kubernetes cluster created.
 - [`istioctl`](https://istio.io/docs/setup/install/istioctl/) (v1.5.4 or later) installed.
 
+## Supported Istio versions
+
+The current known-to-be-stable version of Istio tested in conjunction with Knative is **v1.5.7**.
+Versions in the 1.5 line above **v1.5.4** should generally be fine too.
+Versions above the 1.5 line are under test but have not stabilized yet.
```

Comment:
It's actually 1.4.x, IIUC net-istio bumped stable before fixing the way serving pinned things.

cc @tcnghia @JRBANCEL

Reply:
yea. currently this is 1.4.10. The plan is to bump to Istio 1.7 soon.

Reply:
Adjusted.